Cookbook cleanup: Relocating 'Display content using simple templates' (#764)
ezsystems/developer-documentation: docs/guide/templates.md

In the example above, three different templates are mentioned, two to be used in the full view, and one in the line view. 
Notice that two separate templates are defined for the `article` Content Type. 
They use the same matcher, but will be used in different situations – one when an Article is displayed in the full view, and one in the line view. 
Templates for each of the view types are located in different folders. 
The line template will also make use of a custom controller, while the remaining cases will employ the default one.

##### Full, line and other views

Each Content item can be rendered differently, using different templates, depending on the type of view it is displayed in.
The default, built-in views are:
 
- **full** – used when the Content item is displayed by itself, as a full page
- **line** – used when it is displayed as an item in the list, for example a listing of contents of a folder
- **embed** – used when one Content item is embedded in another, as a block
- **embed-inline** – used when a Content item is embedded inline in another block 

> Other, custom view types can be created, used for example for [embedding one Content item in another](templates.md#embedding-content-items), but only these four have built-in controllers in the system.

Reviewer: ```suggestion
Other, custom view types can be created, used for example for [embedding one Content item in another](#embedding-content-items), but only these four have built-in controllers in the system.
```

> For more details, see [View provider configuration](content_rendering#configuring-views-the-viewprovider).

Reviewer: ```suggestion
For more details, see [View provider configuration](content_rendering.md#configuring-views-the-viewprovider).
```

<a href="{{ path( location ) }}">Some link to a Location</a>
```

> If you don't have the Location object, but only its Location ID, you can generate the URLAlias the following way:

Reviewer: ```suggestion
If you don't have the Location object, but only its ID, you can generate the URL alias the following way:
```
Is that Location ID or a Content ID. From the context I think it is the other but not sure, so I would left ID to be safe.
